base/operators.jl

```diff
@@ -98,8 +100,12 @@ floating-point values generally only needs to define `==`.
 that `hash(x) == hash(y)`.
 
 This typically means that types for which a custom `==` or `isequal` method exists must
-implement a corresponding `hash` method (and vice versa). Collections typically implement
-`isequal` by calling `isequal` recursively on all contents.
+implement a corresponding [`hash`](@ref) method (and vice versa). Collections typically
+implement `isequal` by calling `isequal` recursively on all contents.
+
+Furthermore, `isequal` is linked with [`isless`](@ref), and that they work together to
+define a fixed total ordering. Thus `isequal(x,y)` must imply `!isless(x, y)` and vice
+versa.
```

Comment:
When I read `vice versa` for the first time, I thought of “`!isless(x, y)` implies `isequal(x, y)`”. 😄
Maybe it would be better to write “and `isless(x, y)` must imply `!isequal(x, y)`”.

Reply:
Thanks - I fixed it to read more like the `isless` documentation.